app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,248 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.*;
+
+import java.util.stream.IntStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService 클래스의")
+public class TaskServiceTest {
+    private final Long givenSavedTaskId = 1L;
+    private final Long givenUnsavedTaskId = 100L;
+    private final String givenTaskTitle = "Test";
+    private final String givenModifyTaskTitle = "Modified";
+
+    private final int repeatTime = 2;
+
+    private TaskService taskService;
+    private Task task;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+
+        task = new Task();
+        task.setTitle(givenTaskTitle);
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_getTasks {
+        @Nested
+        @DisplayName("task가 없다면")
```

Comment:
그냥 task가 없는 것이 아니죠. 테스트는 이 프로젝트에 대해 잘 모르는 사람이 읽을 수 있다고 생각하고 섬세하게 작성하는 것이 좋습니다.
```suggestion
        @DisplayName("등록되어 있는 task가 없다면")
```